src/amo/css/ScreenShots.scss

```diff
@@ -1,6 +1,11 @@
 @import "~core/css/inc/mixins";
 @import "~amo/css/inc/vars";
-$screenshot-height: 200px;
+// screenshot-height value was increased for 
+// https://github.com/mozilla/addons-frontend/issues/4887
+// image-height ensures the image is always 20px less than screenshot-height
+// 20px accounts for the approx size of the horizontal scrollbar
```

Comment:
Could you only write `See: https://github.com/mozilla/addons-frontend/issues/4887` as comment? Preferably above the `image-height` var. Thanks!

Reply:
So remove all the other comments?

Reply:
yes, keep it simple :)